common/beerocks/bwl/dummy/ap_wlan_hal_dummy.cpp

```diff
@@ -139,6 +139,7 @@ bool ap_wlan_hal_dummy::sta_deauth(int8_t vap_id, const std::string &mac, uint32
 bool ap_wlan_hal_dummy::sta_bss_steer(const std::string &mac, const std::string &bssid, int chan,
                                       int disassoc_timer, int valid_int)
 {
+    LOG(DEBUG) << "Got steer request for " << mac << " steer to " << bssid;
```

Comment:
If you are about to change test_client_steering_mandate(), then it's incorrect, there is handling of both options - mandate and opportunity. I added handling only for mandate, so for opportunity bwl   will not be called. I could change message check for only one option.

Reply:
I was indeed wrong about the MULTI_AP_POLICY_CONFIG_REQUEST, I don't know how that happened.

But what I meant was to replace the following with a check of bwl log instead of the log in the agent itself.

```sh
    dbg "Confirming Client Steering Request message was received - mandate"
    check docker exec -it repeater1 sh -c \
        'grep -i -q "CLIENT_STEERING_REQUEST_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
    check docker exec -it repeater1 sh -c \
        'grep -i -q "CLIENT_STEERING_REQUEST_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
```

Reply:
Update test scrip to check for bwl message. Updated it to check only for required agent as check for BSSID was added.